### ISSUE TITLE
Permission to get acess to code

### ISSUE THREAD

**Issue by peprahobedadjei** [herrGowtham/Spray-Drying-System-Design-using-MATLAB#1]
Hi @herrGowtham 

I’m Obed from Ireland, and I am currently working on a project titled “AI-Enhanced Self-Updating Digital Twin for Optimised Spray Drying in Dairy Manufacturing.” I came across your PDF and found it very insightful for my project. I would love to explore the possibility of working with your code and collaborating if you’re open to it. Let me know if you would like to share your code with me.

Best regards,
Obed

**Comment by herrGowtham** [comment]
Hello Obed,

I'm sorry for responding so late. Your mail ended up in my spam folder. Are
you still in need of the codes? Do let me know.


Regards,

Gowtham

**Comment by peprahobedadjei** [comment]
Yes, I need it but I would be glad if we could take a bit more to get insight. If possible please whatsapp me on +91 90401 80878.

Kindly send me a message and lets talk. 
Thank you.
